### PR TITLE
Translate ticket merge email templates in French

### PR DESCRIPTION
**Pull request by signed-log**
Found *closed* issue #577 and decided to fix the two missing templates and translate them in French

### CODE REVIEW

**Comment by dominikklein** [comment]
@mgruner Maybe something you can check?

File: app/views/mailer/ticket_update_received_merge/fr.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Un autre ticket a été fusionné dans le ticket (#{ticket.title})
+
+<div>Bonjour #{recipient.firstname},</div>
+<br>
+<div>
+Un autre ticket a été fusionné dans le ticket (#{ticket.title}) by "<b>#{current_user.longname}</b>".
```
Comment (mgruner) [inline app/views/mailer/ticket_update_received_merge/fr.html.erb]: @signed-log is this "by" a typo, perhaps? Could you check please?

Comment (signed-log) [reply app/views/mailer/ticket_update_received_merge/fr.html.erb]: Yes, it's a typo

**Review by mgruner — APPROVED** [review]

**Comment by mgruner** [comment]
Merged, thanks a lot @signed-log!

**Pull request closed without merging**